infra/token-transfer-server/src/shared.js

```diff
@@ -113,6 +114,20 @@ function calculateWithdrawn(transfers) {
   }, BigNumber(0))
 }
 
+const employeeUnlockDate = process.env.EMPLOYEE_UNLOCK_DATE
+  ? moment.utc(process.env.EMPLOYEE_UNLOCK_DATE)
+  : moment.utc('2020-01-01')
```

Comment:
Paranoia: would it be worth adding a check that if NODE_ENV === 'production' then we crash rather than using that default value? Same question for the other constants below.

Reply:
This is the shared file I mentioned in the comment above. Because this is imported by the client you can't do things like exit early because its not a node target. The HTTP bundle has no concept of things like `process.exit()`.